evol/evolution.py

```diff
@@ -147,6 +147,7 @@ def breed(self,
     def mutate(self,
                mutate_function: Callable[..., Any],
                probability: float = 1.0,
+               elitist: bool = False,
```

Comment:
This feels fair. I only wonder if it makes sense not to keep just the best, but instead to maybe keep the best "n" around? I can imagine that with large populations you might want to keep the top 5% around.

Reply:
That does make stuff _way_ more complicated though 😊

Reply:
True. 

I suppose we can always add this feature later if people think it is really required.